### ISSUE TITLE
Set delay to send mouseover events to the server?

### ISSUE THREAD

**Issue by ateucher** [rstudio/leaflet#369]
`shiny::plotOutput` has the `hover` argument, where you can set a delay for sending hover events to the server. Is it possible to do something like this with leaflet? I have a pretty complex multi-polygon map and am using `input$*_shape_mouseover$id` to perform events on hover. But sweeping the mouse over the map runs over many polygons and causes the event to fire many time in a row and it's a bit jarring.

**Comment by timelyportfolio** [comment]
Would Shiny's `debounce`/`throttle` work here?  Here is the [tutorial](https://shiny.rstudio.com/reference/shiny/latest/debounce.html).  I understand I could be missing something so sorry if off topic.

**Comment by ateucher** [comment]
Thanks @timelyportfolio - that looks like it should do exactly what I need! I'll investigate. Closing for now...

**Comment by timelyportfolio** [comment]
@ateucher The only thing that I thought about after I answered was the clarification that this does not slow the stream of messages passed from leaflet.  `throttle` and `debounce` only slow the processing of the messages by Shiny.

**Comment by ateucher** [comment]
Yeah I saw that too. In this case it was exactly what I wanted and it worked perfectly - so thank you very much! 

I can see cases where you would want to slow down the sending of messages from leaflet though, so I will reopen this and let the maintainers close or address as they see appropriate.
